.github/workflows/versions.yaml

```diff
@@ -0,0 +1,35 @@
+name: Upgrade to latest versions
+
+on:
+  schedule:
+    - cron: '37 * * * *'
```

Comment:
isn't every hour too frequent? Checking every day should already be enough wdyt?

Reply:
Updated to run at 13:37 UTC :)